openfe/setup/molecule.py

```diff
@@ -0,0 +1,30 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+from openfe.utils.molhashing import hashmol
+
+
+class Molecule:
+    """Molecule wrapper to provide proper hashing and equality.
+
+    Parameters
```

Comment:
It's a bit cyclic but should we mention that there's an rdkit Mol as an attribute in the class?

Reply:
Yeah, I should probably docstring the property and also add typing info to this -- wanted to get a version out fast. I'll make those changes after lunch.